F1 menu error (I DID NOTHING TO IT)

### Description of the problem

The f1 menu is broken. When you press it nothing happens and you get errors. This can happen as many times as you wish. If a client press f1, they get kicked for errors.
### How I can make the problem happen again

Just by pressing f1.
### Error log pastebin links

No files were generated, however I have the error code here :

[ERROR] lua/includes/extensions/table.lua:99: bad argument #1 to 'pairs' (table expected, got nil)
1. pairs - [C]:-1
   1. unknown - lua/includes/extensions/table.lua:99
   2. f - gamemodes/darkrp/gamemode/libraries/fn.lua:54
      1. condition - gamemodes/darkrp/gamemode/libraries/fn.lua:136
      2. refresh - gamemodes/darkrp/gamemode/modules/f1menu/cl_chatcommandlabel.lua:22
      3. setChatCommand - gamemodes/darkrp/gamemode/modules/f1menu/cl_chatcommandlabel.lua:18
      4. fillLabels - gamemodes/darkrp/gamemode/modules/f1menu/cl_chatcommandspanel.lua:16
         1. refresh - gamemodes/darkrp/gamemode/modules/f1menu/cl_f1menupanel.lua:74
         2. openF1Menu - gamemodes/darkrp/gamemode/modules/f1menu/cl_f1menu.lua:18
         3. unknown - gamemodes/darkrp/gamemode/modules/f1menu/cl_f1menu.lua:28
         4. unknown - gamemodes/darkrp/gamemode/modules/base/cl_gamemode_functions.lua:46

Status
 API
 Training
 Shop
 Blog
 About

--- a/gamemode/modules/base/sh_createitems.lua
+++ b/gamemode/modules/base/sh_createitems.lua
@@ -93,7 +93,7 @@ local function declareTeamCommands(CTeam)
 					fn.Compose{fn.Curry(fn.Eq, 2)(CTeam.NeedsToChangeFrom), plyMeta.Team},
 					fn.If(
 						fn.Curry(istable, 2)(CTeam.NeedToChangeFrom),
-						fn.Compose{fn.Curry(table.HasValue, 2)(CTeam.NeedsToChangeFrom), plyMeta.Team},
+						fn.Compose{fn.Curry(table.HasValue, 2)(CTeam.NeedToChangeFrom), plyMeta.Team},
 						fn.Curry(fn.Const, 2)(true)
 					)()
 				)(),

